.travis.yml

```diff
@@ -1,4 +1,20 @@
-language: java
+language: android
 jdk:
   - openjdk7
   - oraclejdk8
+env:
+  - TEST=java
+  - TEST=android
+matrix:
+  exclude:
+    - jdk: oraclejdk8
+      env: TEST=android
+android:
+  components:
+    - android-15
+    - build-tools-21.1.1
+    - extra-android-m2repository
+    - sys-img-armeabi-v7a-android-15
```

Comment:
Again if it's possible in Travis I'd recommend to use x86 emulator for better performance 😎